wiki/Tournaments/OWC/2013/de.md

```diff
@@ -288,119 +288,119 @@ Das Turnier kam zu einem Ende mit folgenden Platzierungen:
 
 ### Finale
 
-**07\. Dez. 2013 (Samstag)**
+Samstag, 7. Dezember 2013:
 
-| Team A | Score | Team B | History |
-| :-- | :-: | --: | :-- |
-| ![][flag_KR] **Korea** | **6** - 5 | Taiwan ![][flag_TW] | [#1](https://osu.ppy.sh/community/matches/3233030) |
+| Team 1 | | | Team 2 | Match link |
```

Comment:
If I'm correct, this table has **two spaces** in the empty cells. Please recheck and adjust if necessary (the other tables below of course too).

---

Match link may be `Link zum Match` or `Match-Link`

Reply:
You're right, i will adjust this.